example.js

```diff
@@ -0,0 +1,7 @@
+var Micromark = require('.')
+
+var m = new Micromark()
+
+var res = m.write('# Hello!')
```

Comment:
I don't know why but we can't write more than once and continue emitting tokens or maybe that I didn't test it well enough.
Will those two codes do the same thing: `m.write('# '); m.write('Hello!');` and `m.write('# Hello!');` ?

Reply:
Ahh, right. I didn’t check that yet. But correct, I hope to support streaming, so that you could do multiple writes and those examples are indeed equivalent! 👍